tests/test_diff.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import os
+import unittest
+import signac
+from tempfile import TemporaryDirectory
+
+
+class BaseDiffTest(unittest.TestCase):
+
+    project_class = signac.Project
+
+    def setUp(self):
+        self._tmp_dir = TemporaryDirectory(prefix='signac_')
+        self.addCleanup(self._tmp_dir.cleanup)
+        self._tmp_pr = os.path.join(self._tmp_dir.name, 'pr')
+        self._tmp_wd = os.path.join(self._tmp_dir.name, 'wd')
+        os.mkdir(self._tmp_pr)
```

Comment:
I'm confused as to why these extra directories are being made. You can use the `_tmp_dir` as the project root, and the workspace will be created for you by `signac`, so you should be able to remove the above three lines.